docs/test-service-docker.md

```diff
@@ -1,9 +1,18 @@
 # Run flex-plugins e2e tests against locally running docker version of hrm-service
 
-1. Build the docker contianer locally with: `npm run -w hrm-service docker:build`
+## Requirements
+
+Working aws credentials with access to local ssm parameters configured in your terminal
+
+_This probably doesn't work on windows yet_
```

Comment:
Does it require the other PRs to work on Windows?

Reply:
I *think* that the only thing that *may* be broken on windows is [this line](https://github.com/techmatters/flex-plugins/pull/1000/files#diff-2955bcf7fa3fbdea18663408ee5b3916696b47fe6f7e0f9570201176745095d5R10) in the flex side of the the e2e tests setup because of the *nix specific `source` command. That should probably be moved to dotenv eventually.